andela/activo-api
# #158828436 Fix Authentication Decorator

#### What does this PR do?
>- Fix the invalid audience exception thrown when authenticating a user
#### Description of Task to be completed?
*GIVEN*  a valid token is provided
*WHEN* a user makes a request to a protected endpoint
*THEN* the user should be authenticated, and the request should be successful
#### How should this be manually tested?
>- Run `git fetch origin bg-fix-authentication-decorator-158828436
>- Run `git checkout remote/sbg-fix-authentication-decorator-158828436`
>- Run the application `flask run`
>- Login in to `AIS` to get a new token that has the `audience, issuer, and the iat` claims
>- Make a request to a protected endpoint with the new token and the request should be successful
#### Any background context you want to provide?
>- N/A
#### What are the relevant pivotal tracker stories?
>- [158828436](https://www.pivotaltracker.com/n/projects/2170023/stories/158828436)
#### Screenshots (if appropriate)
N/A

#### Questions:
>- N/A
